includes/class-gdpr-requests.php

```diff
@@ -164,7 +164,8 @@ protected function confirm_request( $key ) {
 
 		$user = get_user_by( 'email', $email );
 
-		// phpcs:disable WordPress.VIP.SlowDBQuery.slow_db_query_meta_key
+		// Added VIP equivalent sniff, as per original plugin PHPCS disabled sniff.
```

Comment:
Oh the plugin was already doing this? I was wondering what it was that was causing the warning?

Reply:
Yeah, the original plugin is(, or I think was) following a set standard, which is not identical to ours, but still somehow conforms to VIP afaik. 

Not sure, this plugin is weird in many and various ways.